static/app/views/ddm/layout.tsx

```diff
@@ -111,6 +112,13 @@ export const DDMLayout = memo(() => {
   );
 });
 
+const SplitDivider = styled((props: DividerProps) => (
+  <BaseSplitDivider {...props} icon={<IconGrabbable size="xs" />} />
+))<DividerProps>`
+  height: 16px;
```

Comment:
Would this fit inside of `&[data-slide-direction='updown'] {` in the base class? maybe also as a `space(2)` call